### ISSUE TITLE
Default Version is not a supported version 

### ISSUE THREAD

**Issue by javatlacati** [codewars/codewars.com#2391]
**Describe the bug**
Can't upgrade from Ruby MRI 2.5.0 to MRI 3.0.0

**To Reproduce**
Select a published Kata with MRI 2.5.0, change MRI to 3.0.0 and the error message appears

**Expected behavior**
No error should appear and it should be able to run it using MRI 3.0.0

**Screenshots**
![image](https://user-images.githubusercontent.com/1063901/119362445-513a4900-bc72-11eb-9801-d6d6c8be1028.png)
![image](https://user-images.githubusercontent.com/1063901/119362505-631bec00-bc72-11eb-9e9a-f1e92a09c791.png)

**Comment by XRFXLP** [comment]
Are you sure the problem is not on your side?

I've just updated the kata to Ruby 3.0.0 without any problem.

**Comment by FArekkusu** [comment]
> Are you sure the problem is not on your side?

No, this problem is omnipresent. Changing **only** the default language version errors out.

**Comment by XRFXLP** [comment]
Thanks, but I think that is not a bug, one have to change organisation of test suite.

**Comment by FArekkusu** [comment]
No, updating the language version never succeeds regardless of whether the code is already compatible or not. This has been reported ages ago in #761.
